.github/workflows/docs/check-build-docs

```
@@ -0,0 +1,10 @@
+#!/bin/sh
```

This script is actually failing (but we are swallowing the error from `build-docs`: see the logs.

---

Repaired

---

Thank you for spotting!